docs/current-user-ref.rst

```diff
@@ -4,3 +4,8 @@ CurrentUser
 
 .. autoclass:: canvasapi.current_user.CurrentUser
     :members:
+
+The CurrentUser class being a subclass of the User class, and thus inherits all of the methods of User,
+i.e the User class is the parent class from which methods are inherited to the Child(CurrentUser) class.
+
+See : https://canvas.instructure.com/doc/api/users.html for details
```

Comment:
In the issue, by "provide a link to the `User` class reference page", I meant in the CanvasAPI documentation (as generated from `user-ref.rst`)

You should be able to do that with a link something like this:

```rst
`Documentation for the User class <user-ref.html>`_
```